main.go

```diff
@@ -91,6 +91,9 @@ func (e *errInvalidOutput) Error() string {
 	return fmt.Sprintf("invalid output %s", e.Output)
 }
 
+// Version is the published version of the utility
+var Version string
```

Comment:
Change this to `version` instead of `Version` and then its compatible with goreleaser!